### PR TITLE
added cascade type to profile

### PR DESCRIPTION
**Pull request by mariyazaiceva**

### CODE REVIEW

File: src/main/java/internship/mbicycle/storify/model/Profile.java [hunk 1]
```diff
@@ -25,26 +25,23 @@
...
-    /*Need entities
-
-    @OneToOne(fetch = FetchType.LAZY)
+    @OneToOne(cascade = CascadeType.REFRESH, fetch = FetchType.LAZY)
     @JoinColumn(name = "basket_id")
     @JsonIgnore
     private Basket basket;
 
-    @OneToMany(mappedBy = "profile", fetch = FetchType.LAZY)
+    @OneToMany(cascade = CascadeType.ALL, mappedBy = "profile", fetch = FetchType.LAZY)
```
Comment (Estrin-MBicycle) [inline src/main/java/internship/mbicycle/storify/model/Profile.java:33]: LAZY type is used by default, dont it?

File: src/main/java/internship/mbicycle/storify/model/Profile.java [hunk 2]
```diff
@@ -25,26 +25,23 @@
...
     @JsonIgnore
     private List<Store> stores;
 
-    @ManyToMany(fetch = FetchType.LAZY)
+    @ManyToMany(cascade = {CascadeType.PERSIST, CascadeType.DETACH, CascadeType.REFRESH, CascadeType.MERGE}, fetch = FetchType.LAZY)
```
Comment (Estrin-MBicycle) [inline src/main/java/internship/mbicycle/storify/model/Profile.java:37]: LAZY type is used by default, dont it?

File: src/main/java/internship/mbicycle/storify/model/Profile.java [hunk 3]
```diff
@@ -25,26 +25,23 @@
...
     @JoinTable(name="favorite",
             joinColumns={@JoinColumn(name="profile_id")},
             inverseJoinColumns={@JoinColumn(name="product_id")})
-    @JsonIgnore
     private List<Product> favorite;
 
     @OneToMany(cascade = CascadeType.ALL, mappedBy = "profile", fetch = FetchType.LAZY)
```
Comment (Estrin-MBicycle) [inline src/main/java/internship/mbicycle/storify/model/Profile.java:43]: LAZY type is used by default, dont it?

**Review by Estrin-MBicycle — CHANGES_REQUESTED** [review]

File: src/main/java/internship/mbicycle/storify/model/Profile.java (hunk 3 above)
Comment (mariyazaiceva) [reply src/main/java/internship/mbicycle/storify/model/Profile.java:43]: Yes, lazy

File: src/main/java/internship/mbicycle/storify/model/Profile.java (hunk 2 above)
Comment (mariyazaiceva) [reply src/main/java/internship/mbicycle/storify/model/Profile.java:37]: Yes, lazy

File: src/main/java/internship/mbicycle/storify/model/Profile.java (hunk 1 above)
Comment (mariyazaiceva) [reply src/main/java/internship/mbicycle/storify/model/Profile.java:33]: Yes, lazy

**Pull request merged**